lib/runner/cli/cli.js

```diff
@@ -130,7 +130,7 @@ class CliRunner {
     }
 
     this.testEnv = Utils.isString(this.argv.env) ? this.argv.env : Settings.DEFAULT_ENV;
-    this.testEnvArray = this.testEnv.split(',');
+    this.testEnvArray = this.testEnv.split(' ');
```

Comment:
Test environments are intended to be separated by comma

Reply:
They are still separated by comma.

If you tried to run `nightwatch -e firefox,MicrosoftEdge,default --parallel 30`

`this.testEnv` from the previous line will hold the environments like this `firefox MicrosoftEdge default`

so splitting them with `(',')` do nothing

That's what I was resolving here

Reply:
For me it comes as a comma separated list, am I missing something? Please feel free to correct me, if I am wrong.

<img width="1292" alt="Screenshot 2022-10-13 at 3 18 43 PM" src="https://user-images.githubusercontent.com/2018070/195564316-93fcc932-84b1-4503-a119-ba9a2f13b634.png">

Reply:
Oh, I got the problem. I was using PowerShell and This is happening on Windows PowerShell only!
![image](https://user-images.githubusercontent.com/19888325/195609218-9a71313f-b608-4a22-8c0b-8cedb5dedfa5.png)

I'll close this PR I think that's a different compatibility issue

Reply:
That is interesting, so powershell is basically splitting comma separated arguments, we need to handle this. I'll update your original issue with this finding.